### ISSUE TITLE
Don't allow redundant categories

### ISSUE THREAD

**Issue by volkanto** [alpercun/fancy-macbook-app#20]
Please be careful for the categories while reviewing the PR. I see some redundant categories like `MacOS` or so which are really not necessary and can create a mess on UI, maybe you can predefine the category list or be more careful and put related review comments to clean redundant ones. Thanks in advance.

**Comment by umitcan07** [comment]
Agree, was confused when choosing categories.
Had to look up the website again real quick to see what other apps had.

**Comment by alpercun** [comment]
Yes, according to the applications in general, we will add a categories data such as an issue template that users can refer to. Thank you for this suggestion. A great idea. @volkanto 🎉
